hw/drivers/sensors/bmp388/syscfg.yml

```diff
@@ -0,0 +1,76 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+syscfg.defs:
+    BMP388_INT1_PIN_HOST:
+        description: 'Interrupt pin number on host device connected to INT1 on device'
+        value: 1
+    BMP388_INT1_PIN_DEVICE:
+        description: 'Interrupt pin number 1 or 2 on accelerometer device'
+        value: 1
+    BMP388_INT1_CFG_ACTIVE:
+        description: 'Set 0 for active-low, 1 for active-high'
+        value: 1
+    BMP388_INT_ENABLE:
+        description: 'Enable interrupt support for the BMP388'
+        value: 1
+    BMP388_FIFO_ENABLE:
+        description: 'Enable fifo support for the BMP388'
+        value: 1
+    BMP388_INT_PIN_HOST:
+        description: 'Host interrupt pin number for the BMP388'
+        value: 31
+    BMP388_INT_NUM:
+        description: 'Index of Host interrupt si_ints for the BMP388'
+        value: 0
+    BMP388_SHELL_ITF_NUM:
+        description: 'Shell interface number for the BMP388'
+        value: 0
+    BMP388_SHELL_ITF_TYPE:
+        description: 'Shell interface type for the BMP388'
+        value: 1
+    BMP388_SHELL_CSPIN:
+        description: 'CS pin for BMP388'
+        value : -1
+    BMP388_SHELL_ITF_ADDR:
+        description: 'Slave address for BMP388'
+        value : 0x76
+    BMP388_SHELL_DEV_NAME:
+        description: 'BMP388 Shell device name'
+        value: "\"bmp388_0\""
+    BMP388_CLI:
+        description: 'Enable shell support for the BMP388'
+        value: 0
+    BMP388_NOTIF_STATS:
+        description: 'Enable notification stats'
+        value: 1
```

Comment:
Stats should also be disabled, we don't need them normally. It's more of a debugging feature.